apps/meteor/client/views/omnichannel/currentChats/CurrentChatsRoute.tsx

```diff
@@ -84,19 +101,21 @@ const useQuery: useQueryType = (
 		}
 
 		if (customFields && Object.keys(customFields).length > 0) {
-			query.customFields = JSON.stringify(customFields);
+			const customFieldsQuery: { [key: string]: string } = {};
+			Object.keys(customFields).forEach((key) => {
+				if (customFields[key] !== '') {
+					customFieldsQuery[key] = customFields[key];
+				}
+			});
```

Comment:
Prefer a compact and functional approach here.

```suggestion
			const customFieldsQuery = Object.fromEntries(Object.entries(customFields).filter(([key, value]) => value !== ''));
```